chrome/locale/de/options.dtd

```diff
@@ -1,62 +1,62 @@
-<!ENTITY prefwindow.title "DKIM Verifier Optionen">
+<!ENTITY prefwindow.title "Einstellungen für DKIM Verifier">
 
 
-<!ENTITY general.label "General">
-<!ENTITY general.general.label "General">
-<!ENTITY dkim.enable.label "Überprüfe DKIM Signaturen">
+<!ENTITY general.label "Allgemein">
+<!ENTITY general.general.label "Allgemein">
+<!ENTITY dkim.enable.label "Überprüfe DKIM-Signaturen">
 
 <!ENTITY dns.label "DNS">
 <!ENTITY dns.resolver.label "Resolver">
-<!ENTITY dns.resolver.value.1.label "JavaScript DNS Bibliothek">
+<!ENTITY dns.resolver.value.1.label "JavaScript-DNS-Bibliothek">
 <!ENTITY dns.resolver.value.2.label "libunbound">
-<!ENTITY dns.getNameserversFromOS.label "Lade DNS Server aus den Betriebssystemeinstellungen">
-<!ENTITY dns.nameserver.label "DNS Server (Syntax (EBNF): host[&quot;:&quot;port]{&quot;;&quot;host[&quot;:&quot;port]} )">
+<!ENTITY dns.getNameserversFromOS.label "Lade DNS-Server aus den Einstellungen des Betriebssystems">
+<!ENTITY dns.nameserver.label "DNS-Server (Syntax (EBNF): host[&quot;:&quot;port]{&quot;;&quot;host[&quot;:&quot;port]} )">
 
-<!ENTITY dns.timeout_connect.label "DNS Verbindungsaufbau-Timeout (in Sekunden)">
-<!ENTITY dns.proxy.enable.label "Benutze einen Proxy für die Verbindung zu den DNS Servern">
-<!ENTITY dns.proxy.type.label "Proxy-Type:">
+<!ENTITY dns.timeout_connect.label "DNS-Verbindungs-Timeout (in Sekunden)">
```

Comment:
Note that this change could indeed make sense for the version 4.0.0 (also for the other languages). But for versions before it, it is a little inaccurate.

In 4.0.0, the timeout is now used both for the connection establishing, and reading/writing of data.
Before, this however was only the timeout for the connection establishing. For reading/writing, there was no timeout (if not configured via a separate hidden preference).

Reply:
Ok, I changed this to `Timeout für DNS-Verbindungsaufbau (in Sekunden)` now.